### PR TITLE
Update DeveloperGuide for add property

### LINKED ISSUE #90: Update DeveloperGuide for add property

### PR DESCRIPTION
**Pull request by Virusrwj223**
Fixes #90

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ Format: `addseller n/SELLER_NAME p/SELLER_PHONE_NUMBER e/SELLER_EMAIL`
 * The `SELLER_EMAIL` should follow the format local-part@domain and adhere to the following constraints: The local-part must consist only of alphanumeric characters and allowed special characters, with no special characters at the beginning or end. After the local-part, there must be an '@' followed by the domain name. The domain name consists of domain labels separated by periods. Each domain label must start and end with an alphanumeric character and may contain hyphens in between. Additionally, the domain must end with a domain label that is at least two characters long.
 
 Examples:
-* `addseller n/Mary p/23456789 e/mary@gmail.com` adds a seller whose name is `Mary`, phone number is `23456789` and email is `mary@gmail.com`.
+* `addseller n/Mary p/83456789 e/mary@gmail.com` adds a seller whose name is `Mary`, phone number is `23456789` and email is `mary@gmail.com`.
```
Comment (jolwnn) [inline docs/UserGuide.md]: Theres a second phone number in that sentence that is still 2-sth

**Review by jolwnn — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -193,6 +193,22 @@ Examples:
...
+
+### Adding a property : `addproperty`
+
+Add a specified property into the client book.
+
+Format: `addproperty c/POSTAL_CODE u/UNIT_NUMBER`
+
+* Adds a property with the specified `POSTAL_CODE` and `UNIT_NUMBER`.
+* The `POSTAL_CODE` must be exactly 6 digits with each digit in the range [0-9]. It does not accept any non-integer characters or spaces
+* The `UNIT_NUMBER` comprises of numbers that is delimited by exactly one dash(-). On either side of the dash are numbers comprising of two or more digit. The range of numbers of the left hand side of the dash is [00-148] and the right hand side is [00-111110]. Other than the dash bo other non-integer characters or spaces are accepted
```
Comment (jolwnn) [inline docs/UserGuide.md]: There's a spelling error "bo" in the last line but I guess you can change it in your next pull? or I can remember to change it for you in mine. And also small thing but two or more "digits".

**Review by jolwnn — APPROVED** [review]

File: docs/UserGuide.md (hunk 2 above)
Comment (Virusrwj223) [reply docs/UserGuide.md]: Thanks for spotting that! I'll change it first then do the merge

Comment (jolwnn) [reply docs/UserGuide.md]: Also we are updating with the screenshot of the property list from apollo's pull request right

**Pull request merged**